client/locations-list-map.js

```diff
@@ -66,6 +66,8 @@ let gOpenInfoWindows = [];
 let gLocationsListEntries = [];
 let gLastLocationRendered = -1;
 
+const url = new URL(window.location);
```

Comment:
if you're putting `url` as a global here, can you also refactor the other places we're using it in this module?

Reply:
possibly just assign `searchParams` as a global instead (and do the same)?

Reply:
Also @drwestco added a `FtmUrl` class for browser compat, can you switch to that

Reply:
eslint should have caught this. did it not run for this PR?

Reply:
Yeah, I can clean this up a little more. :)